lib/app-layout/index.tsx

```diff
@@ -115,20 +116,32 @@ export class AppLayout extends Component<Props> {
       </TransitionDelayEnter>
     );
 
+    const hiddenByRevisions = showRevisions ? true : undefined;
+
     return (
-      <div className={mainClasses}>
+      <div
+        className={mainClasses}
+        aria-hidden={isNavigationOpen ? true : undefined}
+      >
         <Suspense fallback={placeholder}>
-          <div className="app-layout__source-column theme-color-bg theme-color-fg">
+          <div
+            aria-hidden={hiddenByRevisions}
+            className="app-layout__source-column theme-color-bg theme-color-fg"
+          >
             <MenuBar />
             <SearchField />
             <NoteList />
             {showSortBar && <SortOrderSelector />}
           </div>
           {editorVisible && (
             <div className="app-layout__note-column theme-color-bg theme-color-fg theme-color-border">
-              <NoteToolbar />
+              <NoteToolbar aria-hidden={hiddenByRevisions} />
               {showRevisions ? (
-                <NotePreview noteId={openedNote} note={openedRevision} />
+                <NotePreview
+                  aria-hidden={hiddenByRevisions}
```

Comment:
why use `aria-hidden` as opposed to `ariaHidden`?

Reply:
Largely for consistency between [DOM elements and React elements](https://reactjs.org/docs/accessibility.html#wai-aria) (i.e. use kebab case in both instances).

That said, I don't have a strong opinion. Happy to change the key for React elements to be camelcase. Would you like me to do that?

Reply:
@codebykat @sandymcfadden Any thoughts? I don't feel super strongly about it.

Reply:
I think the kebab-case is fine, I don't have strong feelings either.